samples/apply-renderers-to-scene-layer/src/main/java/com/esri/arcgismaps/sample/applyrendererstoscenelayer/components/ApplyRenderersToSceneLayerViewModel.kt

```diff
@@ -0,0 +1,244 @@
+/* Copyright 2025 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.esri.arcgismaps.sample.applyrendererstoscenelayer.components
+
+import android.app.Application
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.lifecycle.AndroidViewModel
+import androidx.lifecycle.viewModelScope
+import com.arcgismaps.Color
+import com.arcgismaps.geometry.Point
+import com.arcgismaps.geometry.SpatialReference
+import com.arcgismaps.mapping.ArcGISScene
+import com.arcgismaps.mapping.ArcGISTiledElevationSource
+import com.arcgismaps.mapping.BasemapStyle
+import com.arcgismaps.mapping.Surface
+import com.arcgismaps.mapping.Viewpoint
+import com.arcgismaps.mapping.layers.ArcGISSceneLayer
+import com.arcgismaps.mapping.symbology.ClassBreak
+import com.arcgismaps.mapping.symbology.ClassBreaksRenderer
+import com.arcgismaps.mapping.symbology.ColorMixMode
+import com.arcgismaps.mapping.symbology.MaterialFillSymbolLayer
+import com.arcgismaps.mapping.symbology.MultilayerMeshSymbol
+import com.arcgismaps.mapping.symbology.SimpleRenderer
+import com.arcgismaps.mapping.symbology.SymbolLayerEdges3D
+import com.arcgismaps.mapping.symbology.UniqueValue
+import com.arcgismaps.mapping.symbology.UniqueValueRenderer
+import com.arcgismaps.mapping.view.Camera
+import com.esri.arcgismaps.sample.sampleslib.components.MessageDialogViewModel
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import kotlinx.coroutines.launch
+
+/**
+ * ViewModel for the ApplyRenderersToSceneLayer sample.
+ * Displays a 3D buildings scene layer and allows switching between different renderers.
+ */
+class ApplyRenderersToSceneLayerViewModel(app: Application) : AndroidViewModel(app) {
+    // URLs for the Helsinki buildings scene and world elevation
+    private val helsinkiBuildingsSceneLayerUrl =
+        "https://services.arcgis.com/V6ZHFr6zdgNZuVG0/arcgis/rest/services/Helsinki_buildings/SceneServer"
+    private val worldElevationServiceUrl =
+        "https://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer"
+
+    // ArcGISTiledElevationSource for world elevation
+    private val elevationSource: ArcGISTiledElevationSource by lazy {
+        ArcGISTiledElevationSource(worldElevationServiceUrl)
+    }
```

Comment:
Seems like this also works without a lazy initialization:  
```suggestion
    // ArcGISTiledElevationSource for world elevation
    private val elevationSource = ArcGISTiledElevationSource(worldElevationServiceUrl)
```

Reply:
Interesting. This was created by the AI tool from an existing sample. I thought it was a pattern used in our samples. Good to know it is not a pattern. Updated this and other places.